CODEOWNERS

```diff
@@ -21,8 +21,8 @@ NOTICES     @garyverhaegen-da @aherrmann-da @cocreature @dasormeter
 /compiler/          @hurryabit @cocreature
 /libs-haskell/      @hurryabit @cocreature
 /daml-assistant/    @sofiafaro-da @hurryabit @cocreature
-/daml-script/       @cocreature @hurryabit @aherrmann-da
-/triggers/          @cocreature @hurryabit @aherrmann-da
+/daml-script/       @cocreature @hurryabit @aherrmann-da @stefanobaghino-da
```

Comment:
fwiw while you’re welcome to add yourself to everything don’t feel obligated. So far people mostly added themselves to the things they are familiar with.

Reply:
The intent was mostly to get notified of changes so that I can get accustomed to the new code, I can hardly be defined an owner for most of the things I subscribed myself to. 😛 I'll make the necessary adjustments in the future if it's overkill.